ONSdigital/sdx-downstream
# Retrieve sequence number from sdx-sequence

**Changes**

Sequence id is now retrieved from the sdx-sequence service and incremented on each request.

**How to test**

Test using the standard docker-compose setup and send several requests using the console. Each request should generate separate files within ftp, where previously with new architecture these would have overwritten one another.

**Who can test**

Anyone but @iwootten
